src/main/java/org/jitsi/videobridge/SctpConnection.java

```diff
@@ -195,6 +196,42 @@ private static synchronized int generateDebugId()
     private final Object syncRoot = new Object();
 
     /**
+     * The {@link PacketQueue} instance in which we place packets coming from
+     * the SCTP stack which are to be sent via {@link #transformer}.
+     */
+    private final RawPacketQueue packetQueue;
+
+    /**
+     * The {@link DtlsPacketTransformer} instance which we use to transport
+     * SCTP packets.
+     */
+    private DtlsPacketTransformer transformer = null;
+
+    /**
+     * The instance which we use to handle packets read from
+     * {@link #packetQueue}.
+     */
+    private final PacketQueue.PacketHandler<RawPacket> handler
+        = new PacketQueue.PacketHandler<RawPacket>()
```

Comment:
Don't really like creating anonymous class here, but that's just my opinion. Would prefer to have named class at the bottom, does the same thing.